src/command_modules/azure-cli-network/azure/cli/command_modules/network/_help.py

```diff
@@ -2058,34 +2165,46 @@
           short-summary: Name of resource group. Required when enabling new regions.
           long-summary: When a previously disabled region is enabled to use Network Watcher, a
             Network Watcher resource will be created in this resource group.
+    examples:
```

Comment:
Remove or improve. Suggestion: Set up a watcher on a region containing a specific VNET, by searching for the VNET and extracting the location.

Reply:
I would definitely not remove. This example is actually fine as-is. Network Watcher is weird and an example of how to set it up is good.  I might recommend adding multiple regions to show that you can enable Network Watcher on multiple regions with one command.

Reply:
Left as is.